src/main/service/mediaAccountApplicationService.ts

```diff
@@ -57,9 +57,17 @@ export class MediaAccountApplicationService {
         return null;
     }
 
-    public async rejectApplication(applicationId, adminId): Promise<object | null> {
+    public async rejectApplication(applicationId, adminId, reasons): Promise<object | null> {
         logHelper.writeLog(adminId, 'REJECT_MEDIA_ACCOUNT', applicationId);
-
-        return accountManagementRequests.updateMediaApplicationStatus(applicationId, 'REJECTED');
+        const emailResponse = await accountManagementRequests.sendMediaApplicationRejectionEmail(
+            applicationId,
+            reasons
+        );
+        const updateStatus = accountManagementRequests.updateMediaApplicationStatus(applicationId, 'REJECTED');
```

Comment:
Just checking....if the previous line failed to send e-mail, do we want to update the media application status, or should we just return error without updating the status?

Reply:
Will address this tomorrow morning :)